prov/gni/test/rdm_sr.c

```diff
@@ -271,7 +290,7 @@ void rdm_sr_setup(bool is_noreg, enum fi_progress pm)
 	dgram_should_fail = 0;
 }
 
-void dgram_sr_setup(bool is_noreg, enum fi_progress pm)
+void dgram_sr_setup_api_version_1_0(bool is_noreg, enum fi_progress pm)
```

Comment:
I'd prefer this and the next function to be a single setup function that takes the version as an argument.